lib/adapter.js

```diff
@@ -375,7 +375,16 @@ module.exports = (function() {
 
         // Run Query
         client.query(query.query, query.values, function __CREATE__(err, result) {
-          if(err) return cb(err);
+          if (err) {
+            return cb(err);
+          }
+
+          // Use this in tests to detect whether a test inserted rows into the
+          // database. After you clear the database, reset DBRecordsInserted to
+          // false.
+          if (process.env.NODE_ENV === 'test' && result.rows.length > 0) {
+            global.DBRecordsInserted = true;
```

Comment:
@kevinburkeshyp Have you considered, rather than modifying sails-postgres itself, instead adding a sinon spy at the start of each test, and checking if the spy was called with >0 rows?
It doesn't seem necessary to hack this into sails-postgresql, and it's not this module's concern to tell our tests that something was saved.
Thoughts?